src/components/notification/checker.ts

```diff
@@ -64,13 +76,17 @@ const smtpNotificationInitialChecker = async (data: SMTPData) => {
       subject: subject,
       text: body,
     })
+
+    return 'success'
```

Comment:
is the string return here needed? I think returning nothing is okay here

Reply:
Not required, just for indicating the checker works as expected. For advanced testing it might useful to return `success` instead of returning `undefined`.